src/include/txn.i

```diff
@@ -795,8 +795,9 @@ __wt_txn_read_upd_list(WT_SESSION_IMPL *session, WT_UPDATE *upd, WT_UPDATE **upd
             continue;
         upd_visible = __wt_txn_upd_visible_type(session, upd);
         if (upd_visible == WT_VISIBLE_TRUE) {
-            /* Don't consider TOMBSTONE updates for the history store. */
-            if (type == WT_UPDATE_TOMBSTONE && WT_IS_HS(S2BT(session)))
+            /* Don't consider TOMBSTONE updates for the history store during rollback to stable. */
```

Comment:
I'd rather not capitalize TOMBSTONE in the comment here - comments are supposed to be full English sentences that preferrably don't reference variable names.

Reply:
Fixed it with `tombstone`.